packages/tiler/src/raster.ts

```diff
@@ -7,7 +7,7 @@ export interface TileMaker {
   compose(ctx: TileMakerContext): Promise<{ buffer: Buffer; metrics: Metrics }>;
 }
 
-export type ResizeKernelType = 'nearest' | 'lanczos3' | 'lanczos2';
+export type ResizeKernelType = 'nearest' | 'mitchell' | 'lanczos3' | 'lanczos2';
```

Comment:
we have this type in about 4 places any chance we could merge them somehow?

Reply:
Yeah, we was question about this a lot when doing the change. Might just use the one in config package

Reply:
Turns out we don't install config package in the tiler package. :rofl:

Reply:
lets just do this later.